spec/middleman_prismic_spec.rb

```diff
@@ -1,11 +1,45 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe MiddlemanPrismic do
-  it 'has a version number' do
-    expect(MiddlemanPrismic::VERSION).not_to be nil
+  describe "download from prismic" do
+    it "downloads content from prismic and installs it in the appropriate place" do
+      within_temp_directory do |directory|
+        Capybara::Discoball.spin(FakePrismic) do |server|
+          change_directory_and_configure_with_prismic(directory, server)
+          run_middleman_prismic
+
+          filename = Dir.children("data/prismic/demo-tests/").first
+          document = YAML.load_file(
+            File.join("data/prismic/demo-tests/", filename),
+          )
+
+          expect(document.id).to eq "referoo"
+        end
+      end
+    end
+  end
+
+  def within_temp_directory
+    Dir.mktmpdir do |directory|
+      Dir.chdir(directory) do
+        yield(directory)
+      end
+    end
+  end
```

Comment:
Are the `within_temp_directory` declarations identical?

Could we extract a shared helper or an RSpec trait (similar to `js: true`) for this behavior?